public/app/features/alerting/unified/Silences.tsx

```diff
@@ -31,33 +38,65 @@ const Silences: FC = () => {
     };
   }, [alertManagerSourceName, dispatch]);
 
+  const { result, loading, error } = silences[alertManagerSourceName] || initialAsyncRequestState;
+  const getSilenceById = useCallback((id: string) => result && result.find((silence) => silence.id === id), [result]);
+
   if (!alertManagerSourceName) {
     return <Redirect to="/alerting/silences" />;
   }
-  const { result, loading, error } = silences[alertManagerSourceName] || initialAsyncRequestState;
 
   return (
     <AlertingPageWrapper pageId="silences">
       <Field label="Choose alert manager">
         <AlertManagerPicker current={alertManagerSourceName} onChange={setAlertManagerSourceName} />
       </Field>
-      <br />
-      <br />
+      {isRoot && (
```

Comment:
still should be part of `<SilencesTable />` and not shown if there are no silences. `<NoSilencesCTA />` has it's own new silence button, so two are shown in such case right now